park_api/geodata.py

```diff
@@ -41,6 +41,17 @@ def __init__(self, city):
                 self._process_json(json.load(f))
         except FileNotFoundError:
             self.lots = {}
+        private_file = city[:-3] + ".json"
```

Comment:
I'm guessing this is removing `.def`? Wouldn't this lead to an extra period, which might still work? Or am I miscounting? Nevertheless, it might be easier to understand by just replacing the `def` with `json` in the filename, or even better just the last occurrence of it should any city name contain "def" as well^^